core/contracts/financial-templates/implementation/Liquidatable.sol

```diff
@@ -420,4 +420,8 @@ contract Liquidatable is PricelessPositionManager {
         require(uuid < liquidationArray.length && liquidationArray[uuid].state != Status.Uninitialized);
         return liquidationArray[uuid];
     }
+
+    function _getLiquidationExpiry(LiquidationData storage liquidation) internal view returns (uint) {
```

Comment:
This is good. We don't need to calculate the expiry up front if it's always a function of `liquidationTime` and the `liquidationLiveness`